app/views/documents/form_partials/_validate_document.html.erb

```diff
@@ -0,0 +1,16 @@
+<%= form_with model: [@planning_application, @document], local: true, class: "document", builder: GOVUKDesignSystemFormBuilder::FormBuilder do |form| %>
+  <div class="govuk-form-group" id="validate-document">
+    <% if form.object.errors.any? %>
+      <% form.object.errors.each do |error| %>
+        <span class="govuk-error-message">
+        <span class="govuk-visually-hidden">Error:</span><%= error.type %></span>
+      <% end %>
+    <% end %>
+    <%= form.govuk_radio_buttons_fieldset(:validated, legend: { size: 's', text: 'Is the document valid?' }) do %>
+      <%= form.govuk_radio_button :validated, 'true', label: { text: 'Yes'} %>
+      <%= form.govuk_radio_button :validated, 'false', label: { text: 'No' } do  %>
+        <%= form.govuk_text_area :invalidated_document_reason, label: { text: '' }, hint: { text: 'Describe in full why the document is invalid. This will be sent to the applicant.'}, rows: 3 %>
```

Comment:
I realise it is easier to label the form input as hint text as that's closer to the style, but it's important that the label is in the label tag, for accessibility, useability and it will mean you don't have to do the `find(:css, "#document-invalidated-document-reason-field")` in your tests.

I'd put the hint text into the label text, and figure out a way to override the style if the design insists on a label that doesn't look like a label

Reply:
oh, I did the other suggestion that rhian mentioned - have both and just css the text to be transparent